packages/@react-types/color/src/index.d.ts

```diff
@@ -151,7 +151,11 @@ export interface ColorAreaProps extends ValueBase<string | Color> {
   /** The step value for the xChannel. */
   xChannelStep?: number,
   /** The step value for the yChannel. */
-  yChannelStep?: number
+  yChannelStep?: number,
+  /** The page step size for the xChannel. */
+  xChannelPageStep?: number,
+  /** The page step size for the yChannel. */
+  yChannelPageStep?: number
```

Comment:
What's the use case to configure these?

Reply:
If someone sets a step size bigger than the page step size, then they may want to increase the page step size to compensate.
It might make more sense with HSL/HSB where you can have decimals and you actually want to decrease the page step size.

Reply:
what if we just multiply the step by some amount to find the page step?